breadbox/Dockerfile

```diff
@@ -1,45 +1,17 @@
-FROM us.gcr.io/broad-achilles/depmap-breadbox-base:v2
+FROM us.gcr.io/broad-achilles/depmap-breadbox-base:v2 
```

Comment:
I'm curious how/where this base image is defined. It's probably setting the python version for our docker build. But I haven't been able to find the Dockerfile it's built from. 

It is possible to set the docker python version using pyenv ([example here](https://gist.github.com/jprjr/7667947)), but it's not the most elegant thing. From my understanding, the standard/simpler approach is to set the python version in your base image ([like this](https://github.com/broadinstitute/snp_str_qc/blob/develop/Dockerfile)) and then use pyenv to manage the python version for local development.

Reply:
This was built using the `Dockerfile` in `depmap/depmap-base`. 

It was originally created to speed up building the final image, by starting with a docker image that had things that _rarely_ changed pre-installed. In particular *R and it's dependencies used to take > 30 minutes to install*. 

However, now that we no longer depend on R, it's entirely possible that we could stop using this. I _suspect_ the remaining dependencies are fairly quick to install.